docs/src/architecture.md

```diff
@@ -85,18 +203,13 @@ In a distributed PostgreSQL cluster there can only be a single PostgreSQL
 instance acting as a primary at all times. This means that applications can
 only write inside a single Kubernetes cluster, at any time.
 
-!!! Tip
-    If you are interested in a PostgreSQL architecture where all instances accept writes, 
-    please take a look at  [EDB Distributed Postgres by EDB](https://www.enterprisedb.com/docs/pgd/latest/). 
-    For Kubernetes, EDB Distributed Postgres will have its own Operator, expected later in 2022.
-
 However, for business continuity objectives it is fundamental to:
 
 - reduce global **recovery point objectives** (RPO) by storing PostgreSQL backup data
   in multiple locations, regions and possibly using different providers
-  (**Disaster Recovery**)
+  (DR)
```

Comment:
Using DR has always been a bit strange for me that I'm not used to this terminology, i think it's better explain since the operator it's going to be used for people that may not be used to this term